hack/images/dcos/bootstrap.sh

```diff
@@ -71,6 +71,12 @@ controller_manager_host=${CONTROLLER_MANAGER_HOST:-${default_dns_name}}
 controller_manager_port=${CONTROLLER_MANAGER_PORT:-10252}
 echo "* controller manager: $controller_manager_host:$controller_manager_port"
 
+enable_dns=${ENABLE_DNS:-true}
```

Comment:
Why create a similarly named variable? If it is unset, we can set a default to it: `: ${ENABLE_DNS="true"}`